configure.ac

```diff
@@ -5227,12 +5235,69 @@ if test "x$enable_btls" = "xyes"; then
 	AC_SUBST(BTLS_CMAKE_ARGS)
 
 	AC_DEFINE(HAVE_BTLS, 1, [BoringTls is supported])
+	AC_DEFINE(HAVE_TLS, 1, [Unix Tls is supported])
```

Comment:
This needs to be either renamed into something like `HAVE_UNIX_TLS` or not renamed at all.  I don't have any strong preference on the new name (although I don't understand why it needs to be renamed at all) as long as it clearly reflects the meaning of the constant and does not clash with any "global" things.  Since the name `HAVE_TLS` would suggest to mean _all_ TLS implementations, it cannot be used in this context.

Reply:
Not to mention being confusable for thread-local storage...